src/game/client/gameclient.h

```diff
@@ -156,11 +156,14 @@ class CGameClient : public IGameClient
 		const CNetObj_SpectatorInfo *m_pSpectatorInfo;
 		const CNetObj_SpectatorInfo *m_pPrevSpectatorInfo;
 		const CNetObj_Flag *m_paFlags[2];
+		const CNetObj_RaceFlag *m_paRaceFlags[3];
```

Comment:
`m_apRaceFlags`

`m_paFlags` should also be `m_apFlags` instead, but it's easier to leave that be for now.